test/Driver/cxx_interop.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %swiftc_driver -driver-print-jobs -target x86_64-apple-macosx10.9 %s -enable-experimental-cxx-interop 2>^1 | %FileCheck -check-prefix ENABLE %s
+
+// RUN: %swiftc_driver -driver-print-jobs -target x86_64-apple-macosx10.9 %s -enable-experimental-cxx-interop -experimental-cxx-stdlib libc++ 2>^1 | %FileCheck -check-prefix STDLIB %s
+
+// ENABLE: swift
+// ENABLE: -enable-cxx-interop
+
+// STDLIB: swift
+// STDLIB-DAG: -enable-cxx-interop
+// STDLIB-DAG: -Xcc -stdlib=libc++
```

Comment:
Could you add a negative check on no `-experimental-cxx-stdlib` being passed?  Id like to ensure that the MSVCPRT path is not broken.

Reply:
Done (and discovered what I think is a `FileCheck` bug in the process).